cfe_internal/update/update_policy.cf

```diff
@@ -300,13 +298,106 @@ bundle agent cfe_internal_update_policy_cpv
 bundle agent cfe_internal_update_cmdb
 # @brief Ensure local cache of CMDB data is up to date
 {
+  methods:
+
+    policy_server.enterprise_edition::
+      "cfe_internal_update_cmdb_data_distribution";
+
+      # Don't pull CMDB data on policy_hub self bootstrap because
+      # there will be no cf-serverd listening to serve files yet.
+    enterprise_edition.!(am_policy_hub.bootstrap_mode)::  # ENT-6840
+      "cfe_internal_update_cmdb_data_consumption";
+}
+
+bundle agent cfe_internal_update_cmdb_data_distribution
+# @brief Ensure data is ready for agents to download
+{
+  vars:
+    policy_server.enterprise_edition::
+
+      # The API response for host specific data from cmdb tells us the timestamp of the last data change
+      # We store this timestamp and use it for the next request.
+      "_cmdb_next_request_state_file"
+        string => "$(sys.statedir)/cmdb_next_request_from.dat";
+
+      # If we have the timestamp from a previous response we use it, else we start from 0
+      "_cmdb_previous_next_request_from"
+        string => readfile( $(_cmdb_next_request_state_file), inf ),
+        if => regline( "^\d+$", $(_cmdb_next_request_state_file) );
+
+      "_cmdb_previous_next_request_from"
+        string => "0",
+        unless => regline( "^\d+$", $(_cmdb_next_request_state_file) );
+
+      # We need a script to call that should return the API response
+      "_get_cmdb_data_bin" string => "$(sys.bindir)/get_cmdb.php";
```

Comment:
I don't think the script should be in `/var/cfengine/bin`.

Reply:
Sure, this is just where I put it (manually) while I wait for @aleksandrychev  to give me a final location.

Reply:
Blocked by ENT-7194